themes/uhsg_theme/css/style.css

```diff
@@ -8795,6 +8795,11 @@ input[type="text"].search-form-large__input {
   margin-bottom: 0;
 }
 
+.accordion__item:focus:focus {
+  outline: currentColor thin dashed;
+  outline-offset: 3px;
+}
```

Comment:
I think we should make the focus styling bit more visible. I rarely see a difference.

![Screenshot of comparison between unfocused and focused accordion styling](https://user-images.githubusercontent.com/93661/54915623-4227a700-4f00-11e9-85a6-4409ffd1e723.png)

Reply:
Disregard my comment. I did not restart caches properly after switching the branch.